src/Features/Core/Portable/RemoveUnnecessaryImports/AbstractRemoveUnnecessaryImportsCodeFixProvider.cs

```diff
@@ -39,7 +39,7 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
             return service.RemoveUnnecessaryImportsAsync(document, cancellationToken);
         }
 
-        private class MyCodeAction : CodeAction.DocumentChangeAction
+        private class MyCodeAction : CustomCodeActions.DocumentChangeAction
```

Comment:
`DocumentChangeAction` is an internal nested type on a public type `CodeAction`, which is not accessible in the shared layer. We should eventually delete all such nested internal types inside CodeAction, and switch to shared layer `CustomCodeActions`.